File: flyweight/exercise/Character.java
package flyweight.exercise;

class Character implements TextElement {
    private char character;
    private Font font;
    private Color color;

    public Character(char character, Font font, Color color) {
        this.character = character;
        this.font = font;
        this.color = color;
    }

    @Override
    public void render(int xPosition, int yPosition) {
        // Render the character at position (xPosition, yPosition) with font and color
    }
}

File: C#/Flyweight/Exercise/Character.cs
namespace Flyweight.Exercise;

public sealed class Character : ITextElement
{
    private char _character;
    private Font _font;
    private Color _color;

    public Character(char character, Font font, Color color)
    {
        _character = character;
        _font = font;
        _color = color;
    }

    public void Render(int xPosition, int yPosition)
    {
        // Render the character at position (xPosition, yPosition) with font and color
    }
}
